src/services/user/userService.js

```diff
@@ -0,0 +1,26 @@
+export const UserService = {
+  getUser: async (userId) => {
+    try {
+      const response = await fetch(`http://localhost:3000/user/${userId}`);
+
+      if (!response.ok) {
+        throw new Error('Ocorreu um erro na requisição');
+      }
+
+      const { name, pets } = await response.json();
+
+      if (pets && pets.length > 0) {
+        return pets;
+      }
+      return `${name} não possui pets.`;
```

Comment:
Isso aqui é um problemão

Quando eu tenho `pets` retornamos uma lista de pets num `array`

Quando eu não tenho pets eu retorno uma `string`, meio que estamos enviando uma "mensagem de erro" quando a lista está vazia e isso não é ideal. Lista vazia é um retorno perfeitamente válido

No caso não precisamos desse `if` e basta retornar o valor

```suggestion
        return pets;
```

Reply:
Resolvido